docs/sources/vendor/a10networks/vthunder.md

```diff
@@ -0,0 +1,30 @@
+# a10networks vthunder
+
+## Key facts
+
+* MSG Format based filter
+* Legacy BSD Format default port 514
+
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on  | None                                                                                                    |
+
+## Sourcetypes
+
+| sourcetype     | notes                                                                                                   |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| a10networks:vThunder:cef | None                                                                                          |
+
+## Source
+
+| source         | notes                                                                                                   |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| a10networks:vThunder | None                                                                                              |
+
+### Index Configuration
+
+| key            | source         | index          | notes          |
+|----------------|----------------|----------------|----------------|
+| -| a10networks:vThunder| netwaf         | none           |
```

Comment:
key should be `vendor_product`, like: https://splunk.github.io/splunk-connect-for-syslog/main/sources/vendor/Citrix/netscaler/